sickbeard/properFinder.py

```diff
@@ -265,34 +266,34 @@ def _downloadPropers(self, properList):
             historyResults = main_db_con.select(
                 "SELECT resource FROM history " +
                 "WHERE showid = ? AND season = ? AND episode = ? AND quality = ? AND date >= ? " +
-                "AND action IN (" + ",".join([str(x) for x in Quality.SNATCHED + Quality.DOWNLOADED]) + ")",
+                "AND (action LIKE '%2' OR action LIKE '%4')",
                 [curProper.indexerid, curProper.season, curProper.episode, curProper.quality,
                  historyLimit.strftime(History.date_format)])
 
             # make sure that none of the existing history downloads are the same proper we're trying to download
-            clean_proper_name = self._genericName(helpers.remove_non_release_groups(curProper.name))
+            clean_proper_name = self._genericName(remove_strings(remove_non_release_groups(curProper.name), ['.mkv','.avi','.mp4']))
```

Comment:
Yeah but id rather have like one function that can be Used. For ex call it sanitize_proper() or class ProperParser(). That can be Used System wide. Now you have the risk that here you doe the release goup check, and extention removal. But if you need the same in other parts, it's really Easy to make mistakes. Also I havent worked with it myself, but couldnt we use guessit for Stuff like this? @Labrys opinion?